app/components/shared/sdg/goal_stats_component.rb

```diff
@@ -0,0 +1,15 @@
+class Shared::SDG::GoalStatsComponent < ApplicationComponent
+  with_collection_parameter :goal
+
+  attr_reader :goal
+
+  def initialize(goal:)
+    @goal = goal
+  end
+
+  private
+
+    def approved_budget
```

Comment:
When I read this name I thought it would return something like `Budget.approved.first`. How about naming it `approved_amount`?